### ISSUE TITLE
Redirects 13.6

### ISSUE THREAD

**Issue by kathycarothers** [fecgov/fec-cms#4080]
### Summary

**What we're after:**
We need to put in redirects for pages and files that have been migrated to the CMS recently. These will be the pages from the Policy tab in the redirect file in Google. 

### Related issues

Redirects files from

#4029 (This will include the policy ticket listed and links in the Accessibility tab)

### Completion criteria

- [X] Transition redirects implemented for files and pages migrated
- [x] Redirects tested after @patphongs reviews, merges and deploys
- [x] www redirects added in Wagtail if needed -will tackle this in a future ticket
- [x] Take-down ticket put in for the redirected files - will tackle this in a future ticket

### Future work

Take-down ticket.

**Comment by kathycarothers** [comment]
We decided not to include the accessibility documents at this time. We are still waiting for some of the annual reports before we can do the redirects for this section. 

Note,  I have highlighted in orange in the Policy tab on redirects we are going to hold off for now on.
